frontend/src/components/ManageSchema.vue

```diff
@@ -0,0 +1,144 @@
+<!--
+ Copyright (c) 2021 - for information on the respective copyright owner
+ see the NOTICE file and/or the repository at
+ https://github.com/hyperledger-labs/organizational-agent
+
+ SPDX-License-Identifier: Apache-2.0
+-->
+<template>
+  <v-container>
+    <v-card class="mx-auto">
+      <v-card-title class="bg-light">
+        <span>{{ schema.label }}</span>
+        <v-layout align-end justify-end>
+          <v-btn
+              depressed
+              color="red"
+              icon
+              :disabled="schema.isReadOnly"
+              @click="deleteSchema"
+          >
+            <v-icon dark>$vuetify.icons.delete</v-icon>
+          </v-btn>
+        </v-layout>
+      </v-card-title>
+      <v-container>
+        <v-list-item class="mt-4">
+          <v-text-field
+              id="schemaId"
+              v-model="schema.schemaId"
+              readonly
+              outlined
+              dense
+              label="Schema ID"
+              :append-icon="'$vuetify.icons.copy'"
+              @click:append="copySchemaId"
+          ></v-text-field>
+        </v-list-item>
+
+      </v-container>
+      <v-container>
+        <v-tabs>
+          <v-tab href="#schema-attributes">Schema Attributes</v-tab>
+          <v-tab-item transition=false value="schema-attributes">
+            <v-card flat class="mt-2">
+              <v-list-item
+                  v-for="attribute in schema.schemaAttributeNames"
+                  :key="attribute.id"
+              >
+                <p class="grey--text text--darken-2 font-weight-medium">
+                  {{ attribute }}
+                </p>
+              </v-list-item>
+            </v-card>
+          </v-tab-item>
+          <v-tab href="#credential-definitions">Credential Definitions</v-tab>
+          <v-tab-item transition=false value="credential-definitions">
+            <v-card flat class="mt-2">
+              <credential-definitions
+                  :schema="schema"
+                  :credentialDefinitions="schema.credentialDefinitions"
+                  @changed="onChanged"/>
+            </v-card>
+          </v-tab-item>
+          <v-tab href="#trusted-issuers">Trusted Issuers</v-tab>
```

Comment:
Managing trusted issuers within the Credential Management view feels strange. I'd hide this tab.

Reply:
trusted issuer hidden on Credential management, and in Settings, it is only available on added/imported schemas.